commit 7d213541cb194e845fc01e00d7d9bb93d87a6f57
Author: Diego Nieto Cid <dnietoc@gmail.com>
Date:   Sun May 31 16:53:42 2026 -0300

    hurd: define SO_TIMESTAMP in socket.h
    
    * sysdeps/mach/hurd/bits/socket.h: add SO_TIMESTAMP enum entry
      and define corresponding preprocessor macro.
    Message-ID: <20260531195342.1633-1-dnietoc@gmail.com>

diff --git a/sysdeps/mach/hurd/bits/socket.h b/sysdeps/mach/hurd/bits/socket.h
index fb73dea063..15733df61a 100644
--- a/sysdeps/mach/hurd/bits/socket.h
+++ b/sysdeps/mach/hurd/bits/socket.h
@@ -364,6 +364,8 @@ enum
 #define SO_OOBINLINE SO_OOBINLINE
     SO_REUSEPORT = 0x0200,	/* Allow local address and port reuse.  */
 #define SO_REUSEPORT SO_REUSEPORT
+    SO_TIMESTAMP = 0x0400,	/* Timestamp received dgram traffic */
+#define SO_TIMESTAMP SO_TIMESTAMP
     SO_SNDBUF = 0x1001,		/* Send buffer size.  */
 #define SO_SNDBUF SO_SNDBUF
     SO_RCVBUF = 0x1002,		/* Receive buffer.  */
